test/expand-envs.spec.ts

```diff
@@ -7,12 +7,15 @@ describe('expandEnvs', (): void => {
     notvar: 'this is not used',
     dollar: 'money',
     PING: 'PONG',
-    IP1: '127.0.0.1'
+    IP1: '127.0.0.1',
+    WHO: 'World'
   }
-  const args = ['notvar', '$dollar', '\\$notvar', '-4', '$PING', '$IP1', '\\$IP1', '$NONEXIST']
-  const argsExpanded = ['notvar', 'money', '\\$notvar', '-4', 'PONG', '127.0.0.1', '\\$IP1', '$NONEXIST']
 
-  it('should replace environment variables in args', (): void => {
+  // eslint-disable-next-line no-template-curly-in-string
```

Comment:
I needed to disable the rule for this line, otherwise I could not commit my test string.